#! /bin/sh /usr/share/dpatch/dpatch-run
## 14-enable-portaudio-by-default.dpatch by  <tsmithe@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Try to ensure audio always works by defaulting to PortAudio enabled.

@DPATCH@
diff -urNad trunk~/mscore/mscore/preferences.cpp trunk/mscore/mscore/preferences.cpp
--- trunk~/mscore/mscore/preferences.cpp	2009-10-08 17:15:15.170997010 +0100
+++ trunk/mscore/mscore/preferences.cpp	2009-10-08 17:19:34.013497946 +0100
@@ -167,7 +167,7 @@
 #else
       useAlsaAudio       = true;
       useJackAudio       = false;
-      usePortaudioAudio  = false;
+      usePortaudioAudio  = true;
       useJackMidi        = false;
 #endif
       alsaDevice         = "default";
@@ -382,7 +382,7 @@
       useAlsaAudio       = s.value("useAlsaAudio", true).toBool();
       useJackAudio       = s.value("useJackAudio", false).toBool();
       useJackMidi        = s.value("useJackMidi",  false).toBool();
-      usePortaudioAudio  = s.value("usePortaudioAudio", false).toBool();
+      usePortaudioAudio  = s.value("usePortaudioAudio", true).toBool();
 #endif
 
       alsaDevice         = s.value("alsaDevice", "default").toString();
